File: Basic/Form a number divisible by 3 using array digits.cpp
//{ Driver Code Starts
// Initial Template for C++

#include <bits/stdc++.h>
using namespace std;

// } Driver Code Ends
// User function Template for C++

class Solution
{
  public:
    int isPossible(int N, int arr[])
    {
        // code here
        int somma = 0;
        
        for (int i=0; i<N; i++)
        {
            somma += arr[i] %3;
        }
        
        if (somma %3 == 0) return true;
        return false;
    }
};

//{ Driver Code Starts.

int main() {
    int t;
    cin >> t;
    while (t--) {
        int N;
        cin >> N;
        int arr[N];
        for (int i = 0; i < N; i++) cin >> arr[i];

        Solution ob;
        cout << ob.isPossible(N, arr) << endl;
    }
    return 0;
}
// } Driver Code Ends

File: Easy/Form a number divisible by 3 using array digits.py
#User function Template for python3

class Solution:
    def isPossible(self, N, arr):
        # code here
        digitSum = 0
        
        for it in arr:
            while it > 0:
                digitSum += it % 10
                it //= 10
        
        if digitSum % 3 == 0:
            return 1
        
        return 0
                


#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input())
        arr = input().split()
        for i in range(N):
            arr[i] = int(arr[i])
        
        ob = Solution()
        print(ob.isPossible(N, arr))
# } Driver Code Ends